### ISSUE TITLE
How to use ParallelSampledSoftmaxLoss in expert_util.py

### ISSUE THREAD

**Issue by rivaldinho** [tensorflow/tensor2tensor#174]
I'm training the language model for 1B words benchmark on its full-size vocabulary(790000+), need sampled softmax loss to speedup the training process. I guess ParallelSampledSoftmaxLoss in expert_utils.py is actually what I need, but not sure how to use this function. Which part of code should I modify to call this function and how to modify that? Maybe top_sharded in modality.py? or something in t2t_model.py?
Can you guys show an example to make use of it instead of current full softmax?
Thanks a lot!

**Comment by lukaszkaiser** [comment]
We experimented a fair bit with sampled softmax. It gave worse results than using subword tokens every time and required a lot of maintenance and looking into memory problems. That's why we removed that support from SymbolModality. You can rebuild it in your own modality, but since it's consistently bad, we will probably not commit to maintaining it forever unless there are some compelling results.
